hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt

```diff
@@ -23,12 +23,14 @@ data class HmppsSqsProperties(
     val dlqMaxReceiveCount: Int = 5,
     val visibilityTimeout: Int = 30,
     val errorVisibilityTimeout: Int = 0,
+    val propagateTracing: Boolean = false,
   )
 
   data class TopicConfig(
     val arn: String = "",
     val accessKeyId: String = "",
     val secretAccessKey: String = "",
+    val propagateTracing: Boolean = false,
```

Comment:
The main change - you have to turn on the new tracing headers if you want them. This will allow us to experiment with turning them on before committing.